sc/src/test/java/uk/ac/ebi/atlas/monitoring/JsonBuildVersionControllerWIT.java

```diff
@@ -37,10 +38,11 @@ public void setUp() {
     @Test
     public void requestReturnsBuildInfo() throws Exception {
         mockMvc.perform(get("/json/build"))
-                .andExpect(status().isOk())
+                .andExpect(status().isOk()).andDo(print())
```

Comment:
I guess this is just to aid in debugging?

Reply:
Oops, yes it was! I'll remove it 🙂